crates/gpui/src/platform/blade/blade_atlas.rs

```diff
@@ -42,10 +43,11 @@ impl BladeAtlasState {
 pub struct BladeTextureInfo {
     pub size: gpu::Extent,
     pub raw_view: gpu::TextureView,
+    pub msaa_view: Option<gpu::TextureView>,
 }
 
 impl BladeAtlas {
-    pub(crate) fn new(gpu: &Arc<gpu::Context>) -> Self {
+    pub(crate) fn new(gpu: &Arc<gpu::Context>, sample_count: u32) -> Self {
```

Comment:
This is a bit wasteful. We only want MSAA atlas tiles for things that actually require MSAA.
The atlas contains texture of different formats via `AtlasTextureKind`.
A general way to implement it is to turn `BladeAtlasStorage ` into basically a `HashMap<BladeAtlasKey, AtlasTextureList<BladeAtlasTexture>>`, where `BladeAtlasKey` would be a combination of both `AtlasTextureKind` and the sample count.

A less general way would be to just make it explicitly `path_sample_count`, so that it only affects the rasterized path.
Honestly, I think this is a better way due to its simplicity.

Reply:
Makes sense.

I have changed only Path kind to create msaa_texutre